backend/app/config/config.js

```diff
@@ -0,0 +1,4 @@
+//secret word for validating tokens
+module.exports = {
+    secret: 'safeTraceEnigma245237401975'  //this value must be changed for every different app
```

Comment:
Can you please document this in the README, so that it's not just "hidden" in the code?